untie/src/untie.cpp

```diff
@@ -0,0 +1,19 @@
+
+#include "untie.h"
+#include <iostream>
+
+namespace knot {
+
+Untie::Untie() {
+    m_engine = std::make_unique<knot::Engine>();
+    m_window = m_engine->get_window_module();
+}
+
+void Untie::run() {
+    std::cout << "here" << std::endl;
+    while (m_window->is_open()) {
```

Comment:
Maybe we want to abstract this `is_open` to the engine, so we can also have flags which can dictate if the main loop should break.

Reply:
Clion disable "Add to targets"

Reply:
Engine::is_open